### PR TITLE
Add image with NeptuneUI

### PR DESCRIPTION
**Pull request by DunderRoffe**
Adds an image that includes NeptuneUI. In order to keep a qt free version of the Pelux reference this is done as a new image.

### CODE REVIEW

File: recipes-qt/automotive/neptune-ui/neptune.service [hunk 1]
```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Neptune
+After=dbus.service dbus-session.service systemd-user-sessions.service
+
+[Service]
+ExecStart=/usr/bin/appman -r -c /opt/am/config.yaml -c am-config.yaml --dbus none Main.qml -platform eglfs
```
Comment (GordanM) [inline recipes-qt/automotive/neptune-ui/neptune.service]: NeptuneUI should have dbus enabled by default: 

`ExecStart=/usr/bin/appman -r -c /opt/am/config.yaml --dbus session -c am-config.yaml Main.qml`

File: recipes-qt/automotive/neptune-ui/neptune.service [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+Restart=on-failure
+WorkingDirectory=/opt/neptune
+Environment=QT_IM_MODULE=qtvirtualkeyboard
```
Comment (GordanM) [inline recipes-qt/automotive/neptune-ui/neptune.service]: This is only true if the qtvirtualkeyboard is a part of PELUX.

File: recipes-qt/automotive/neptune-ui/neptune.service (hunk 1 above)
Comment (GordanM) [inline recipes-qt/automotive/neptune-ui/neptune.service]: Also, I'll get back to you on another curiosity I have regarding the service startup.

File: recipes-qt/automotive/neptune-ui/neptune.service (hunk 2 above)
Comment (DunderRoffe) [inline recipes-qt/automotive/neptune-ui/neptune.service]: qtvirtualkeyboard is an RDEPENDS on the neptuneui recipe in boot2qt

**Comment by DunderRoffe** [comment]
(assuming you are speeking of the .bbappend) Yes I agree.

**Pull request merged**